ADDRESSING.md

```diff
@@ -80,13 +83,13 @@ Outside of browser context, and in cases when site isolation does not matter (se
 
 Examples:
 
-    https://gateway.ipfs.io/ipfs/bafybeiemxf5abjwjbikoz4mc3a3dla6ual3jsgpdr4cjr3oz3evfyavhwq/wiki/Vincent_van_Gogh.html
-    https://gateway.ipfs.io/ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/wiki/Mars.html
-    https://gateway.ipfs.io/ipns/tr.wikipedia-on-ipfs.org/wiki/Anasayfa.html
+    https://dweb.link/ipfs/bafybeiemxf5abjwjbikoz4mc3a3dla6ual3jsgpdr4cjr3oz3evfyavhwq/wiki/Vincent_van_Gogh.html
+    https://dweb.link/ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/wiki/Mars.html
+    https://dweb.link/ipns/tr.wikipedia-on-ipfs.org/wiki/Anasayfa.html
```

Comment:
The dnslink is mutable, recently changed to version without `.html` suffix, and may change over time, so better to use generic landing page in this example:
```suggestion
    https://dweb.link/ipns/tr.wikipedia-on-ipfs.org/wiki/
```